hwdb.d/60-evdev.hwdb

```diff
@@ -732,3 +732,13 @@ evdev:input:b0003v172Fp0047*
 evdev:input:b0003v172Fp0505*
  EVDEV_ABS_00=::160
  EVDEV_ABS_01=::160
+ 
+#########################################
+# Positivo-Vaio
+#########################################
+# Vaio FE14
+evdev:name:SYNA3602:00 0911:5288 Touchpad:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:pnVJFE41F11*:pvr*
+ EVDEV_ABS_00=::28
+ EVDEV_ABS_01=::27
+ EVDEV_ABS_35=::28
+ EVDEV_ABS_36=::27
```

Comment:
This file is ordered alphabetically by vendor, can you move this block up please to where it belongs, thanks